### DISCUSSION TITLE
Table of contents - INDEX

### DISCUSSION THREAD

**Discussion by lkabi** [CMSgov/price-transparency-guide#452] [Q&A] [answered]
What is the 'Index' on Table of contents File name ?
naming standard for the table-of-contents file: <YYYY-MM-DD>_<payer or issuer name>_**index**.<file extension>

Should it be a sequence number of ToC?

**Comment by shaselton-usds** [comment, marked as answer]
The "_index" is static in the naming convention. For example `2020-08-07_cms_index.json`

There was a [webinar on Nov 30th](https://www.cms.gov/healthplan-price-transparency/resources/webinars) that covered the naming convention for table of content files. You can view the recording or download the slides to get a better idea on the process.

**Reply by lkabi** [reply]
Thanks for the prompt response.
